lib/ddtrace/context.rb

```diff
@@ -283,7 +301,7 @@ def reset(options = {})
       @overflow = false
     end
 
-    def set_current_span(span)
+    def current_span=(span)
```

Comment:
Just a reminder to ourselves: this is a breaking API change/

Reply:
It's probably worth creating an alias for backwards compatibility at this time.

Reply:
Shouldn't be: this method was marked `private`.